plugins/auth-core.yml

```diff
@@ -1,11 +1,11 @@
 id: auth-core
-name: Auth-core
-description: Add the core functionality for authentication and authorization to your apps
+name: NestJS Auth Module
+description: Add the core functionality for authentication and authorization to your service. This plugin must be installed with one of the Auth Provider plugins.
 repo: amplication/plugins
 npm: '@amplication/plugin-auth-core'
 icon: auth-core.png
 github: https://github.com/amplication/plugins/tree/master/plugins/auth-core
 website: https://github.com/amplication/plugins/tree/master/plugins/auth-core
 type: official
-categories: [security]
+categories: [core]
```

Comment:
@yuval-hazaz I added this category after a conversation with @barshimi 
We are thinking about having a "core" category, and plugins in this category will be excluded from the plugin order functionality. This may solve the problem of counting on the user to install every core plugin before another plugin in the same category.